tests/test_admin.py

```diff
@@ -58,6 +72,26 @@ def test_ls(monkeypatch, client):
         ],
     )
 
+    assert result.output == (
```

Comment:
Put it in separate test function, with relevant function name corresponds the test goal.